### PR TITLE
[idl] Add Graph::applyEdgeLeafConstraints.

### PR DESCRIPTION
**Pull request by florent-lamiraux**

### CODE REVIEW

File: src/graph.impl.cc [hunk 1]
```diff
@@ -674,6 +674,52 @@ namespace hpp {
...
+          }
+
+	  ULong size = (ULong) config->size ();
+	  hpp::floatSeq* q_ptr = new hpp::floatSeq ();
+	  q_ptr->length (size);
+
+	  for (std::size_t i=0; i<size; ++i) {
+	    (*q_ptr) [(ULong) i] = (*config) [i];
+	  }
+	  output = q_ptr;
```
Comment (jmirabel) [inline src/graph.impl.cc]: output = vectorToFloatSeq (*config);

File: src/hpp/corbaserver/manipulation/constraint_graph.py [hunk 2]
```diff
@@ -512,6 +512,21 @@ def  applyNodeConstraints (self, node,  input) :
     #
     #  Compute a configuration in the destination node of the edge,
     #  reachable from qFrom.
+    def applyEdgeLeafConstraints (self, edge, qfrom, input) :
```
Comment (jmirabel) [inline src/hpp/corbaserver/manipulation/constraint_graph.py]: I think the documentation is not correct here. The applied constraints do not contain the constraints of the destination state, do they ?

File: src/graph.impl.cc (hunk 1 above)
Comment (florent-lamiraux) [reply src/graph.impl.cc]: Fixed by f85bdfe.

File: src/hpp/corbaserver/manipulation/constraint_graph.py (hunk 2 above)
Comment (florent-lamiraux) [reply src/hpp/corbaserver/manipulation/constraint_graph.py]: Fixed by f85bdfe.

**Pull request merged**